Userland/Libraries/LibAudio/FlacLoader.cpp

```diff
@@ -978,7 +978,7 @@ i64 sign_extend(u32 n, u8 size)
 {
     // negative
     if ((n & (1 << (size - 1))) > 0) {
-        return static_cast<i64>(n | (0xffffffff << size));
+        return static_cast<i64>(n | (0xffffffffffffffffLL << size));
```

Comment:
maybe just `~0LL`?
(opinion)

Reply:
I think it's more important to have an explicit amount of f's here and let the compiler warn if our literal suffix is not large enough.